suite-common/graph/src/graphDataFetching.ts

```diff
@@ -262,12 +271,21 @@ export const getMultipleAccountBalanceHistoryWithFiat = async ({
                     fiatCurrency,
                     forceRefetch,
                     isElectrumBackend,
-                }).then(res => {
-                    if (res === null)
-                        throw new Error(`Unable to fetch fiat rates for defined timestamps.`);
-
-                    return [coin, res] as const;
-                }),
+                })
+                    .then(res => {
+                        if (res === null)
+                            throw new Error(`Unable to fetch fiat rates for defined timestamps.`);
+
+                        return [coin, res] as const;
+                    })
+                    .catch(error => {
+                        console.error(
+                            `Unable to fetch GRAPH fiat rates ${fiatCurrency} for ${coin}:`,
```

Comment:
Should the error message her and in then clause in `if (res === null)` be different?

Reply:
That error message from `if (res === null)` will be inherited here so I think it's not necessary to change it.